server/subscribe.go

```diff
@@ -280,6 +281,16 @@ func (p *Plugin) getSubscriptionTemplatesForInstance(instanceID types.ID) (*Temp
 	return subs, nil
 }
 
+func (p *Plugin) getSubscriptionTemplatesByID(instanceID types.ID, templateID string) (*SubscriptionTemplate, error) {
```

Comment:
```suggestion
func (p *Plugin) getSubscriptionTemplatesByID(instanceID, templateID types.ID) (*SubscriptionTemplate, error) {
```
Shouldn't templateID be also `types.ID` everywhere?

Reply:
I need the template Id here as a string to save it in KVstore. We usually have keys for KV store as string that's why i have taken it as a string.

Reply:
But isn't `types.ID` also a string?